### PR TITLE
Update allowedPeers.mainnet.ts

### PR DESCRIPTION
**Pull request by jessandrich**
## Motivation

Added my peerid

## Change Summary

Describe the changes being made in 1-2 concise sentences.

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [ ] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [ ] PR has a [changeset](https://github.com/farcasterxyz/hub-monorepo/blob/main/CONTRIBUTING.md#35-adding-changesets)
- [ ] PR has been tagged with a change label(s) (i.e. documentation, feature, bugfix, or chore)
- [ ] PR does not require changes to the [protocol](https://github.com/farcasterxyz/protocol)
- [ ] All [commits have been signed](https://github.com/farcasterxyz/hub-monorepo/blob/main/CONTRIBUTING.md#22-signing-commits)

## Additional Context

If this is a relatively large or complex change, provide more details here that will help reviewers

---

## PR-Codex overview
This PR adds a new allowed peer to the `mainnet` network in the `allowedPeers.mainnet.ts` file.

### Detailed summary
- A new allowed peer has been added to the `mainnet` network.
- The peer's address is `12D3KooWGU1ebBRwdTBrfb2jQniip7FsWfr22PJhZQwwAf4d8jgX`.
- The peer's username is `@jessandrich`.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/hubble/src/allowedPeers.mainnet.ts [hunk 1]
```diff
@@ -46,4 +46,5 @@ export const MAINNET_ALLOWED_PEERS = [
   '12D3KooWRDvjugV5fhkTA2BBmrLXmKyBXLe4V7YJyMXyLQGj1PAM', // @rysiman
   '12D3KooWNLshuXk4x2gacFTMEJjMGGFxPjpHg5KcV88KoqJP7dDR', // @bhgomes
   '12D3KooWS39hrt41rMuD5ZfPFMxUVycxVwz7w15CxZVhxP4AZDGr', // @rozhnovskiyigor
+  '12D3KooWGU1ebBRwdTBrfb2jQniip7FsWfr22PJhZQwwAf4d8jgX', // @jessandrich
```
Comment (varunsrin) [inline apps/hubble/src/allowedPeers.mainnet.ts:49]: can you please add your farcaster id here so that we can reach you easily?

Comment (deodad) [reply apps/hubble/src/allowedPeers.mainnet.ts:49]: I couldn't find jessandrich, is this your Farcaster name?

Comment (jessandrich) [reply apps/hubble/src/allowedPeers.mainnet.ts:49]: No. I still haven't received an invite from Varun. My telegram @Gvozdaryova43654

**Comment by varunsrin** [comment]
added in a batch pr here: https://github.com/farcasterxyz/hub-monorepo/pull/1072

**Pull request closed without merging**